cmd/juno/juno.go

```diff
@@ -82,8 +82,12 @@ func main() {
 
 	config := new(node.Config)
 	cmd := NewCmd(config, func(cmd *cobra.Command, _ []string) error {
-		fmt.Printf("%s\n\n", greeting)
-
+		log, err := utils.NewZapLogger(utils.INFO, true)
+		if err != nil {
+			return err
+		}
+		// Log the greeting and version
+		log.Info(fmt.Sprintf(greeting, Version))
```

Comment:
I don't want to instantiate a logger here, please change it to the following 
```suggestion
		fmt.Sprintf(greeting, Version))
```

once you have made this change I will approve